openapi/paths/aml-checks@{id}.yaml

```
@@ -0,0 +1,29 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve results of a Customer's AML check
+  operationId: GetAmlCheck
+  x-sdk-operation-name: get
+  description: |-
+    Retrieve the results of a Customer's AML check.
+    Customer metadata and an array of matching AML hits is returned.
```

issue: [start all operations with an active verb](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations)
```suggestion
    Retrieves the results of a customer's AML check.
    Customer metadata and an array of matching AML hits is returned.
```